code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -1,28 +1,32 @@
 /obj/item/weapon/storage/bible
 	name = "bible"
-	desc = "Apply to head repeatedly."
-	icon_state ="bible"
+	desc = "Apply to person's head repeatedly."
+	icon_state = "bible"
 	throw_speed = 1
 	throw_range = 5
 	w_class = 3.0
+	force = 2.5 //A big book, solely used for non-Chaplains trying to use it on people
 	flags = FPRINT
+	attack_verb = list("whack", "slap", "slam")
 	var/mob/affecting = null
 	var/deity_name = "Christ"
 
 	autoignition_temperature = 522 // Kelvin
 	fire_fuel = 2
 
-	suicide_act(mob/user)
-		viewers(user) << "<span class='danger'>[user] is farting lightly on the [src.name]! It looks like \he's  trying to commit suicide!</span>"
-		return (user.death(1))
-
+/obj/item/weapon/storage/bible/suicide_act(mob/living/user)
+	viewers(user) << "<span class='danger'>[user] is farting on \the [src]! It looks like \he's trying to commit suicide!</span>"

```

Comment:
What kind of logic is that?

Issue with `viewers()` is that blind people will *also* see it.